src/main/scala/view/module/cactus/Panes.scala

```diff
@@ -79,23 +79,23 @@ class MainPane(controller: CactusController) extends ScalaFXPane:
 
   private val topPosition: Int = 0
 
-  private val leftPosition: Int                                    = 0
-  private val topLeftCorner: ViewPosition                          = ViewPosition(topPosition, leftPosition)
-  private def paneCenter: ViewPosition                                 = ViewPosition(paneWidth, paneHeight) / 2
+  private val leftPosition: Int                                   = 0
+  private val topLeftCorner: ViewPosition                         = ViewPosition(topPosition, leftPosition)
+  private def paneCenter: ViewPosition                            = ViewPosition(paneWidth, paneHeight) / 2
+  private val realPlayer: Player                                  = controller.players(0)
```

Comment:
Maybe it's better to have a method in the Model that tells the View who the real player is. The decision of who the human player is, is more of an aspect of the model.